lib/process-services/src/lib/task-list/components/task-filters.component.spec.ts

```diff
@@ -105,6 +105,18 @@ describe('TaskFiltersComponent', () => {
         component.ngOnInit();
     });
 
+    it('should be able to call create default task filters api if there are no filters available on given appId', async () => {
```

Comment:
Should call the API to create the default task filters when no task filters exist